src/networking/PersonService.ts

```diff
@@ -0,0 +1,27 @@
+import { NetworkService } from "./NetworkService";
+import { COLLECTION_NAME } from "./PopulationOptions";
+import Person from "../models/Person";
+
+export class PersonService {
+  networkService: NetworkService = NetworkService.getInstance();
```

Comment:
So if I understood your comments on the base service, this call will either use the cached instance or make a new one correct?

Reply:
Yes. Since every model will have its own Model Service (e.g. PersonService) and every Model Service has a NetworkService... just have them all use the same one, it's cheaper that way.